docs/guides/prompt-manager/api-access.mdx

```diff
@@ -257,58 +211,94 @@ The response contains the document details along with its configuration.
 ```json
 {
   "id": "document-id",
-  "name": "Document Name",
+  "documentUuid": "document-uuid",
+  "path": "path/to/document",
   "content": "Document content",
+  "resolvedContent": "Document content without comments",
+  "contentHash": "content-hash",
+  "commitId": "commit-id",
+  "deletedAt": "deleted-at",
+  "createdAt": "created-at",
+  "updatedAt": "updated-at",
+  "mergedAt": "merged-at",
+  "projectId": "project-id",
   "config": {
     "provider": "Provider name",
     "model": "Model name"
   }
 }
 ```
 
-**Example Request:**
+#### 4. Evaluate a Conversation
+
+Evaluate a conversation using configured evaluations.
+
+**Endpoint:** `POST /conversations/{conversationUuid}/evaluate`
+
+**Path Parameters:**
+
+- `conversationUuid`: UUID of the conversation to evaluate
+
+**Request Body:**
 
-```bash
-curl -X GET "https://gateway.latitude.so/api/v2/projects/123/versions/live/documents/path/to/document" \
-     -H "Authorization: Bearer YOUR_API_KEY"
+```json
+{
+  "evaluationUuids": ["evaluation-uuid-1", "evaluation-uuid-2"] // optional, defaults to all evaluations connected to the conversation prompt
+}
 ```
 
-**Example Response:**
+**Response:**
 
 ```json
 {
-  "id": "document-id",
-  "name": "Document Name",
-  "content": "Document content",
-  "config": {
-    "provider": "Provider name",
-    "model": "Model name"
-  }
+  "evaluations": ["evaluation-uuid-1", "evaluation-uuid-2"] // array of evaluation UUIDs that will be run
 }
 ```
 
-#### 4. Evaluate a Conversation
+#### 5. Create Log Entry
 
-Evaluate a conversation using configured evaluations.
+Create a log entry for a document.
 
-**Endpoint:** `POST /conversations/{conversationUuid}/evaluate`
+**Endpoint:** `POST /projects/{projectId}/versions/{versionUuid}/documents/logs`
 
 **Path Parameters:**
 
-- `conversationUuid`: UUID of the conversation to evaluate
+- `projectId`: Your project ID (required)
+- `versionUuid`: Version UUID (required, optional for SDK's defaults to 'live')
 
 **Request Body:**
 
 ```json
 {
-  "evaluationUuids": ["evaluation-uuid-1", "evaluation-uuid-2"] // optional, defaults to all evaluations connected to the conversation prompt
+  "path": "path/to/document",
+  "messages": [
+    {
+      "role": "user" | "system" | "assistant",
+        {
+          "type": "text", //atm only text is supported
+          "content": string
+        }
+    }
```

Comment:
this is not true you can pass any message accepted by [vercel's ai sdk](https://sdk.vercel.ai/docs/reference/ai-sdk-core/stream-text#api-signature) (which follows OAI's format)

Reply:
fixxed with new commit